src/auth/logout.rs

```diff
@@ -0,0 +1,64 @@
+use crate::auth::session::UserSession;
+use crate::error;
+use crate::generate_endpoint;
+use actix_web::HttpResponse;
+use anyhow::{bail, Result};
+use tracing::info;
+
+pub async fn end_user_session(email: &str) -> Result<()> {
+    if let Some(session) = UserSession::fetch_by_email(email.to_string()).await {
+        session.delete().await?;
+        println!("Deleted the session");
+        Ok(())
+    } else {
+        bail!("No such session!")
+    }
+}
+
+pub async fn clear_tokens(email: &str) -> Result<()> {
+    if let Some(mut session) = UserSession::fetch_by_email(email.to_string()).await {
+        session.access_token = String::new();
```

Comment:
Maybe use `String::clear` here?
```suggestion
        session.access_token.clear();
```

Reply:
As i said in my comment this method is not doing anything as we delete the session afterwards, so probably dont need to think about it to much